src/test/groovy/net/rubygrapefruit/platform/file/AbstractFileEventsTest.groovy

```diff
@@ -336,7 +336,8 @@ abstract class AbstractFileEventsTest extends Specification {
 
     def "can start and stop watching directory while changes are being made to its contents"() {
         given:
-        def numberOfParallelWriters = 100
+        def numberOfParallelWritersPerWatchedDirectory = 100
```

Comment:
That is currently too high. Shall we lower it to 25? I can polish the test in another PR.